tests/codeception/acceptance/OpportunityContentCest.php

```diff
@@ -43,21 +46,28 @@ public function testRelatedOpportunities(AcceptanceTester $I) {
     $terms = $this->createTaxonomyTerms($I, [
       'Foo Bar Baz',
     ], 'su_opportunity_type');
-
+    /** @var \Drupal\taxonomy\TermInterface $term */
+    $term = reset($terms);
     $I->createEntity([
       'type' => 'su_opportunity',
       'title' => 'Foo Bar',
-      'su_opp_type' => reset($terms)->id(),
+      'su_opp_type' => $term->id(),
     ]);
+    $dimensions = $this->createTaxonomyTerms($I, [
+      'Foo Bar Baz',
+    ], 'su_opportunity_dimension');
     $node = $I->createEntity([
       'type' => 'su_opportunity',
       'title' => 'Bar Foo',
-      'su_opp_type' => reset($terms)->id(),
+      'su_opp_type' => $term->id(),
+      'su_opp_dimension' => reset($dimensions)->id(),
```

Comment:
reset($terms)->id() was changed to $term = reset($terms) and $term->id(). Do you want to do the same thing with reset($dimensions) since it's also used twice?

Reply:
meh. its all the same. and its just a test so less important